jekyll/_cci2/google-auth.md

```diff
@@ -29,7 +29,7 @@ to deploy your application to GCP products.
 
 ## Steps
 
-### Choosing a Base Image for the Deploy Job
```

Comment:
There are many reasons to install the Google Cloud SDK; not all of them involve a deploy job.

Reply:
It's worth noting that the Google Cloud SDK is pre-installed in our Android images.